js/ui/map.js

```diff
@@ -287,6 +287,27 @@ util.extend(Map.prototype, /** @lends Map.prototype */{
     },
 
     /**
+     * Set constraint on the map's geographical bounds.
```

Comment:
I'm not exactly even sure of the behavior. We may never have tried to precisely define it before. Is that description based on trying it out?